Add a changelog

knothed opened the pull request.

knothed committed:
Add a changelog for all former releases

fredpi commented:
Thanks for the PR 👍

A few notes:
• I don‘t think we should list changes of the first release ;)
• Please add a section „Unreleased“ on top with the corresponding subtitles (Added, Changed...).
• We should reference issues and authors in the changelog
• We should mention the changelog in the CONTRIBUTING.md

knothed commented:
@fredpi What should "Unreleased" mean? Should this section be updated each time stable is updated? Does this make sense?

knothed commented:
Also, I think it makes sense to outline what the capabilities of the first release were. Especially as they are currently the key part of the project.

knothed committed:
Update changelog

• Add author names
• Add current changes in "Unreleased" section. Attention: This includes changes from #19 and #20.

knothed committed:
Change changelog entry for #19 (work/test-availability)
